server/db/seeds/01_users.js

```diff
@@ -0,0 +1,49 @@
+exports.seed = function (knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users_table').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('users_table').insert([
+        { user_id: 0505,
```

Comment:
You could leave user_id's out since the primary key will be autogenerated and autoincremented.

Reply:
Yes I am aware PKs will autogenerate IDs. When I seeded, I wanted to give the username a user_id that I know so I assigned birthdates as user_id's. For example, user_id: 0320, first_name: "Romeo" - 0320 is my birthdate. The reason for this is for testing purposes due to not being able to render the names when I first set this up.